src/Controller/AppbannersController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Drupal\appbanners\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Component\Utility\Html;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+/**
+ * Provides route responses for the Example module.
+ */
+class AppbannersController extends ControllerBase {
+
+  /**
+   * Page callback for manifest.json.
+   */
+  public function manifestPage() {
+
+    $config = \Drupal::config('appbanners.settings');
```

Comment:
ideally we'd use dependency injection for some of this stuff, but we can worry about that later - more valuable to get a working D8 version first